enterprise/cypher/acceptance-spec-suite/src/test/resources/cypher/features/PatternPredicates.feature

```diff
@@ -304,6 +304,23 @@ Feature: PatternPredicates
       | (:A)  |
     And no side effects
 
+  Scenario: Using a optional match after aggregation and before an aggregation
```

Comment:
I think this scenario should be moved to : https://github.com/SaschaPeukert/neo4j/blob/6af43b6ff2a740b37f71b9c75a1318809f30a825/enterprise/cypher/acceptance-spec-suite/src/test/resources/cypher/features/OptionalMatchAcceptance.feature
(1) there is no pattern predicate in the scenario, e.g., `WHERE <pattern>`
(2) because that is an Optional Match-specific feature file